src/components/pagination/pagination.stories.tsx

```diff
@@ -30,7 +30,7 @@ export const Primary: Story = {
           pageSize={pageSize}
```

Comment:
Let's increase the width of the wrapper to at least `900px` to show all controls.